### PR TITLE
Add blog article about import suggestions

### PR DESCRIPTION
**Pull request by julienrf**

### CODE REVIEW

**Comment by julienrf** [comment]
It looks like the following:

Screenshot_2020-04-28 Import Suggestions in Scala 3

File: blog/_posts/2020-05-05-scala-3-import-suggestions.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+title: Import Suggestions in Scala 3
+---
+
+Implicits let the compiler “write” significant parts of a program for you.
+For instance, the compiler can summon JSON serializers and deserializers for
+a complete type hierarchy.
+
+However, working with implicits can be a difficult experience. Thankfully, the
+Scala 3 compiler dramatically
+improves the quality of the error messages shown in case of missing implicit so
```
Comment (odersky) [inline blog/_posts/2020-05-05-scala-3-import-suggestions.md:14]: missing implicits

File: blog/_posts/2020-05-05-scala-3-import-suggestions.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+The error message now shows how far the compiler went by chaining
+implicit definitions, and where it eventually stopped because an
+implicit argument could not be found. In our case, we see that the
+compiler tried the definition `orderList` but then didn’t find an
+implicit `Order[Foo]`. So, we know that to fix the problem we need
+to implement an implicit `Order[Foo]`.
+
+> For the record, the idea of showing the complete chain of implicits was
+> pioneered by Torsten Schmits in the [splain] compiler plugin, which is
+> available in Scala 2.
```
[the remark this answers is not in the document]
Comment (julienrf) [reply blog/_posts/2020-05-05-scala-3-import-suggestions.md:142]: It’s to “emulate” a side note (like we can usually do in markup languages like [paradox](https://developer.lightbend.com/docs/paradox/current/directives/callouts.html) or [ornate](https://szeiger.github.io/ornate-doc/markdown.html#blockquoteattributes)).

File: blog/_posts/2020-05-05-scala-3-import-suggestions.md [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+// The applicative type class (its actual definition does not matter for the example)
+trait Applicative[F[_]]
+~~~
+
+Let’s assume that an implicit instance of type `Traverse[List]` and an implicit instance
+of type `Applicative[Option]` are defined in an object `Implicits`:
+
+~~~
+object Implicits {
+  implicit def traverseList: Traverse[List] = ???
```
Comment (odersky) [inline blog/_posts/2020-05-05-scala-3-import-suggestions.md:222]: Should this use `given` syntax, since this is about Scala 3?

**Comment by julienrf** [comment]
Thanks a lot for your review. I’ve addressed your feedback in the last two commits.

**Comment by SethTisue** [comment]
What Dotty version do people need to be using in order to try it?

**Comment by julienrf** [comment]
I recommend the latest (0.24.0-RC1). I’ve added a sentence to mention that.

**Pull request merged**